### PR TITLE
Flavor performance

### PR DESCRIPTION
**Pull request by JosiahRegencia**
This is for issue #39 I uploaded in Slack the run time it took to save the data to the database. I'll remove those print codes once you approve.

### CODE REVIEW

File: savant/app/record.py [hunk 1]
```diff
@@ -1,55 +1,91 @@
...
+
+    user_start_time = timeit.default_timer()
+    for user in get_scanned.users:
+        if check_exists.user_exists(user[0], user[1], user[2], user[3], user[4], user[5], user[6])[0][0] == 'FALSE':
+            cur.callproc('store_users', (user[0], user[1], user[2], user[3], user[4], user[5], user[6],))
+        else:
+            continue
+    user_elapsed = timeit.default_timer() - user_start_time
+
+    print "Debs: ", debs_elapsed
```
Comment (greenmoss) [inline savant/app/record.py]: Please change these print statements to `log.debug`

File: savant/app/results.py [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
-
 @app.route('/doveps/api/flavor/create/', methods=['POST'])
 def create_flavors():
+    start_time = timeit.default_timer()
     filenames = request.files.getlist('files[]')
+    elapsed = timeit.default_timer() - start_time
     get_scanned.get_items(filenames)
     record.record_base_flavors()
 
+    print "Elapsed Time: ", elapsed
```
Comment (greenmoss) [inline savant/app/results.py]: Same as above; please use `log.debug`

**Review by greenmoss — CHANGES_REQUESTED** [review]

**Comment by JosiahRegencia** [comment]
noticed something off when testing manually, still fixing

**Comment by JosiahRegencia** [comment]
Just an update, the screenshot I showed previously wasn't really exact. Although from 32 seconds average I was able to trim it down to 10 seconds. Because of many data in debs (362 rows) it takes 8 seconds to complete. For groups (51 rows) 1 second and 0.5 seconds for shadow and users (25 rows each)

**Comment by JosiahRegencia** [comment]
This last push, flavor creation performs below 1 second. I'll upload a demo later to show how fast it is.

**Comment by JosiahRegencia** [comment]
Did a demo on the updated flavor performance. Here it is: https://youtu.be/WLX7IBnkOVM

**Review by greenmoss — APPROVED** [review]

**Pull request merged**